fe/fe-core/src/main/java/org/apache/doris/nereids/rules/exploration/mv/InitMaterializationContextHook.java

```diff
@@ -78,7 +78,8 @@ public void initMaterializationContext(CascadesContext cascadesContext) {
      * @param cascadesContext current cascadesContext in the planner
      */
     protected void doInitMaterializationContext(CascadesContext cascadesContext) {
-        TableCollectorContext collectorContext = new TableCollectorContext(Sets.newHashSet(), true);
+        TableCollectorContext collectorContext = new TableCollectorContext(Sets.newHashSet(),
+                true, true);
```

Comment:
ditto

Reply:
For query plan, this should not extend, we only get avalible mvs by one level table or view. this is for performance